packages/ember-glimmer/lib/environment.js

```diff
@@ -51,9 +51,11 @@ class PropertyReference {
 }
 
 import { default as concat } from './helpers/concat';
+import { default as inlineIf } from './helpers/inline-if';
 
 const helpers = {
-  concat
+  concat,
+  if: inlineIf
```

Comment:
I can quickly follow this PR up with one adding support for inline unless
if that's OK?
On 17 Feb 2016 6:47 p.m., "Godfrey Chan" <notifications@github.com> wrote:

> In packages/ember-glimmer/lib/environment.js
> <https://github.com/emberjs/ember.js/pull/12920#discussion_r53210816>:
>
> >
> >  const helpers = {
> > -  concat
> > +  concat,
> > +  if: inlineIf
>
> We have an inlineUnless helper in Ember too [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/emberjs/ember.js/pull/12920/files#r53210816>.
>